### ISSUE TITLE
Nginx Proxy Manager: redirecting issue

### ISSUE THREAD

**Issue by aeternum1986** [searxng/searxng#1253]
**Version of SearXNG, commit number if you are using on master branch and stipulate if you forked SearXNG**

**How did you install SearXNG?**

`docker run --rm              -d -p ${PORT}:8080              -v "${PWD}/searxng:/etc/searxng"      -e "BASE_URL=http://my.domain.com"        -e "INSTANCE_NAME=aeternum"            searxng/searxng`

**What happened?**

When I try to search on my searx instance through firefox, it will redirect to "localhost:8080". I am using nginx proxy manager, but they said it's not NPM causing the issue. What should happen is that it should stay on the domain i set up.

**How To Reproduce**

[Nginx Proxy Manager](https://nginxproxymanager.com/) sitting in front of SearXNG as a reverse proxy.

**Expected behavior**

**Screenshots & Logs**

**Additional context**

**Comment by mrpaulblack** [comment]
Update: On matrix we found out that this is not a redirecting issue with SearXNG core itself, but an issue with nginx proxy manager (~~npm~~) sitting in front of SearXNG as a reverse proxy.

**Comment by return42** [comment]
Please do not use *npm* for a nginx proxy manager setup (npm confused me in this context) .. npm is commonly used for the nodejs package manager .. to complete the confusion there is also a https://www.npmjs.com/package/nginx at npm ;-)

BTW: can we close this issue?

**Comment by mrpaulblack** [comment]
This is not a SearXNG issue; Closing. (But please post a solution for your issue here as a comment, if you find it :+1: )

**Comment by raviwarrier** [comment]
Is this issue closed? I just installed searxng and have a similar problem. I tried saving preferences while I was on my raspberrypi ip (http://192.168.1.x:8180) and upon saving the preferences, the URL reverts to http://localhost:8180. Before seeing this issue, I thought maybe nginx proxy manager might help, but from https://search.mydomain.com it still reverted to http://localhost:8180 upon saving preferences.

So, whether on internal ip or public url, I face the same issue... any suggestions?

Nevermind. Just saw the solution of BASE_URL. Thanks and sorry for the trouble. Leaving this post for future reference.
